public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -475,6 +475,16 @@ export function buildGridItemForLibPanel(panel: PanelModel) {
   });
 }
 
+const hoverHeaderOffsetBehavior = (grid: DashboardGridItem) => {
```

Comment:
Can we extract this behavior to a different file? Just to remove noise from this one which is quite critical for Scene creation and it is frequently revisited :)